tubular/nominal.py

```diff
@@ -978,13 +998,13 @@ class OneHotEncodingTransformer(BaseNominalTransformer, OneHotEncoder):
 
     def __init__(
         self,
-        columns=None,
-        separator="_",
-        drop_original=False,
-        copy=True,
-        verbose=False,
-        **kwargs,
-    ):
+        columns: str | list[str] | None = None,
```

Comment:
The functionality of the None option here isn't in the docstring - please can you quickly add something to say that in this case it will transform all of the columns.

Reply:
This actually needs doing in a few places so happy to leave if you don't want to address this in this PR

Reply:
I've added the type to the docstring. BUT long term I really don't see much use in keeping type annotation in the code AND in the docs - let the Sphix documentation build extract the typehints and add them to the docs. See [this plugin](https://github.com/agronholm/sphinx-autodoc-typehints).